core/src/main/scala/akka/persistence/cassandra/query/EventsByTagPublisher.scala

```diff
@@ -263,12 +263,12 @@ private[query] class EventsByTagPublisher(
     case ReplayDone(retrievedCount, deliveredCount, seqN, highest, mightBeMore) =>
       if (log.isDebugEnabled)
         log.debug(
-          s"${backtrackingLog}query chunk done for tag [{}], timBucket [{}], count [{}] of [{}]",
+          s"${backtrackingLog}query chunk done for tag [{}], timBucket [{}], count [{}] of [{}], mightBeMore [$mightBeMore]",
           tag, currTimeBucket, deliveredCount, retrievedCount
         )
       seqNumbers = seqN
       currOffset = highest
-      replayCountZero = retrievedCount == 0
+      replayCountZero = backtrackingMode == NoBacktracking && retrievedCount == 0
```

Comment:
I'm not sure why this change would help. I thought initially that replayCountZero being false (due to duplicate event found) caused lookingfordelayed to loop.